From f255a609b33e2bc1245d4d6895c177bc2b1bfeb9 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Tue, 30 Jun 2015 13:32:59 +0800
Subject: [PATCH] MLK-11202: dtb: Disable lcdif in csi dtb

iMX7D DDR3 ARM2 board LCDIF pin confilict with CSI,
so disable LCDIF in csi dtb.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/boot/dts/imx7d-19x19-ddr3-arm2-csi.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2-csi.dts b/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2-csi.dts
index bb0a09b..facec64 100644
--- a/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2-csi.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-ddr3-arm2-csi.dts
@@ -8,6 +8,10 @@
 
 #include "imx7d-19x19-ddr3-arm2.dts"
 
+&lcdif {
+	status = "disabled";
+};
+
 &csi1 {
 	status = "okay";
 
-- 
1.8.0

